numba/np/arraymath.py

```diff
@@ -1165,6 +1167,75 @@ def impl(a):
 )
 
 
+@register_jitable
+def _isclose_item(x, y, rtol, atol, equal_nan):
+    if np.isnan(x) and np.isnan(y):
+        return equal_nan
+    elif np.isinf(x) and np.isinf(y):
+        return (x > 0) == (y > 0)
+    elif np.isinf(x) or np.isinf(y):
+        return False
+    else:
+        return abs(x - y) <= atol + rtol * abs(y)
+
+
+@overload(np.isclose)
+def isclose(a, b, rtol=1e-05, atol=1e-08, equal_nan=False):
+    if not(type_can_asarray(a) and type_can_asarray(b)):
+        raise TypingError("Inputs for `np.isclose` must be array-like.")
+
+    if isinstance(a, types.Array) and isinstance(b, types.Number):
+        def isclose_impl(a, b, rtol=1e-05, atol=1e-08, equal_nan=False):
+            x = a.reshape(-1)
+            y = b
+            out = np.zeros(len(x), np.bool_)
+            for i in range(len(out)):
+                out[i] = _isclose_item(x[i], y, rtol, atol, equal_nan)
+            return out.reshape(a.shape)
+
+    elif isinstance(a, types.Number) and isinstance(b, types.Array):
+        def isclose_impl(a, b, rtol=1e-05, atol=1e-08, equal_nan=False):
+            x = a
+            y = b.reshape(-1)
+            out = np.zeros(len(y), np.bool_)
+            for i in range(len(out)):
+                out[i] = _isclose_item(x, y[i], rtol, atol, equal_nan)
+            return out.reshape(b.shape)
+
+    elif isinstance(a, types.Array) and isinstance(b, types.Array):
+        m = max(a.ndim, b.ndim)
+        tup_init = (0,) * m
+
+        def isclose_impl(a, b, rtol=1e-05, atol=1e-08, equal_nan=False):
+            # Broadcast arrays of different types - cannot use
+            # np.broadcast_arrays for that
+            # this can be replaced by np.broadcast_shapes once #7437
```

Comment:
It looks like that PR is merged. https://github.com/numba/numba/pull/8003

Can you update this section?

Reply:
I see your comment in this PR. Could you update this comment to reflect that it cannot actually be used until the minimum Numpy version is increased?